draft-pauly-add-requirements.md

```diff
@@ -186,68 +186,73 @@ Using a content-provider's encrypted resolver can also provide several privacy a
+Encrypted resolver discovery mechanisms can either add new parameters to existing provisioning
 mechanisms (adding necessary information to use DoT or DoH to options in DHCP, RAs, or IKEv2) or else provide a way
 to communicate with a provisioned unencrypted DNS resolver and discover the equivalent or associated encrypted
 DNS resolver.
 
-For the second group, mechanisms additionally need to provide some information about the applicability and capabilities
-of encrypted resolvers. This information could either be provisioned,
+Domain-specific encrypted resolver discovery mechanisms additionally need to provide some information about the
+applicability and capabilities of encrypted resolvers. This information could either be provisioned,
 or be discovered based on clients actively trying to access content.
 
 # Privacy and security requirements {#priv-sec}
 
-Since one of the main reasons for enabling encrypted DNS on clients is to improve the privacy and security properties
-of DNS queries, discovery mechanisms must be designed carefully to not weaken a client's security or privacy.
+A primary goal of encrypted DNS is improving the privacy and security of DNS queries and answers in the presence
+of malicious attackers. Such attackers are assumed to interfere with or otherwise impede DNS traffic and corresponding
+discovery mechanisms, in any way possible, in an attempt to prevent usage of encrypted DNS. For example, an attacker
+may block connectivity to a resolver chosen by the user. Clients MUST treat the network as such an attacker unless given
+some means of authenticating or otherwise trusting the network.
 
-- Clients are always responsible for selecting which resolver(s) to use. Standards describing resolver discovery
-mechanisms must not place any requirements on clients to select some resolvers instead of other resolvers.
+Given this type of attacker, discovery mechanisms must be designed carefully to not worsen a client's security or
+privacy posture in such networks. In particular, attackers must not be able to:
 
-- User or administrator preference must always be able to override any discovery mechanism. Networks
-may block connectivity to the resolver chosen by the user, but that does not mean that systems should ignore user
-preference.
+- Redirect DNS traffic to themselves.
 
-- Automatic discovery mechanisms must be tied to an entity that is already otherwise involved in providing
-service to the client, such as: a network provider, a VPN server, a content provider being accessed, or
-a server that the client has manually configured.
+- Override or adversely influence client resolver selection by users or administrators.
 
-- The relationship between a discovered resolver and the client-known entity must be securely authenticated.
+- Cause clients to use a discovered resolver which has no trust association with a client-known entity.
```

Comment:
I think it's still important to include the term "authentication" here?

Reply:
Hmm, let me take another stab at this.